### PR TITLE
missing_formula: update for Pillow's brew demise.

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-----

CC @fxcoudert.

Ref: https://github.com/Homebrew/homebrew-core/pull/17341

### CODE REVIEW

**Comment by ilovezfs** [comment]
pillow is now deleted.

**Comment by DomT4** [comment]
Alright, tweaked this PR.

File: Library/Homebrew/missing_formula.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ def blacklisted_reason(name)
             #{Formatter.url("https://pip.readthedocs.io/en/stable/installing/")}
           EOS
         when "pil" then <<-EOS.undent
-          Instead of PIL, consider `pip install pillow` or `brew install Homebrew/science/pillow`.
+          Instead of PIL, consider `pip install pillow`.
```
Comment (ilovezfs) [inline Library/Homebrew/missing_formula.rb]: probably should say pip2?

Comment (DomT4) [reply Library/Homebrew/missing_formula.rb]: Oh, yeah, I guess so these days.

**Comment by ilovezfs** [comment]
:pill: :o:

**Comment by DomT4** [comment]
Ta ❤️

**Pull request merged**